Formula/n98-magerun.rb

```diff
@@ -0,0 +1,15 @@
+class N98Magerun < Formula
+  desc "The swiss army knife for Magento developers, sysadmins and devops."
```

Comment:
You can shorten this a bit to `Swiss army knife for Magento developers, sysadmins and devops`.